src/static/js/modules/jquery/cf_notifier.js

```diff
@@ -7,8 +7,11 @@
 var $ = require( 'jquery' );
 var handlebars = require( 'handlebars' );
 
-var _notifierTemplate = '<div class="cf-notification cf-notification__{{ state }}" style="display: none;">' +
-  '<span class="cf-notification_icon cf-notification_icon__{{ state }} cf-icon cf-icon-{{ icon }}-round"></span>' +
+var _notifierTemplate = '<div class="cf-notification cf-notification__{{ state }}" ' +
+                             'style="display: none;">' +
+  '<span class="cf-notification_icon ' +
```

Comment:
Yep, that is the plan going forward as I take this out of FJ and into CF (I'm about mid-way done but haven't had time to finish).